Disable the doc generation to work around the bug tracked in:
https://savannah.gnu.org/bugs/index.php?65892

TODO: Drop this patch, once the bug is fixed.
--- a/Makefile.am	2024-09-29 13:41:23.017935137 +0200
+++ b/Makefile.am	2024-09-29 14:05:22.251911243 +0200
@@ -85,7 +85,6 @@ include $(top_srcdir)/examples/automake.
 include $(top_srcdir)/src/automake.mk
 include $(top_srcdir)/utilities/automake.mk
 include $(top_srcdir)/tests/automake.mk
-include $(top_srcdir)/doc/automake.mk
 include $(top_srcdir)/Windows/automake.mk
 
 if WITH_PERL_MODULE
